libs/store/src/lib/payloads/reducers.ts

```diff
@@ -0,0 +1,55 @@
+import {
+  AppTournamentId,
+  AppTournament,
+  AppRaceId,
+  AppRace,
+  AppPlayerId,
+  AppPlayer,
+  AppLeaderboard,
+  AppPlayerLogId,
+  AppPlayerLog,
+  AppErrorLog,
+  AppErrorTimestamp,
+} from '@razor/models';
+
+//reducer Playloads
+export type addTournamentReducerPayload = {
```

Comment:
Please have all `types` in `PascalCase`; not `camelCase`.

Reply:
changed in 7b3383b84c4b1723ebff6302c51e69b2535326f5 & 9d9abb3b34f8680fbfaadb6ffbcdaa1043ca0d28